interview-questions.md

```diff
@@ -10,58 +10,67 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is `this` in JavaScript?
```

Comment:
Your answers are spot on. For your interview questions, please add more of your experience with the answers.  Tell the story of one of the challenges you did to learn this concept. These interview questions are opportunities to practice sharing your thoughts with an interviewer/future clients.